server/service/src/test_helpers.rs

```diff
@@ -17,6 +17,7 @@ pub(crate) struct ServiceTestContext {
     #[allow(dead_code)]
     pub(crate) connection: StorageConnection,
     pub(crate) service_provider: Arc<ServiceProvider>,
+    #[allow(dead_code)]
```

Comment:
#[test] might be better

Reply:
The module of the ServiceTestContext is already `#[cfg(test)]`. This was an issue for the `memory` tests because I disabled the processor tests for them and then this field became unused. (Tried to mark it with `not(memory)` but this had lots of other issues...)